database/couchdb/seedCouchdb.js

```diff
@@ -0,0 +1,29 @@
+const username = 'root';
+const password = 'password';
+const couchdbURL = `http://${username}:${password}@localhost:5984`;
+const nano = require('nano')(couchdbURL);
+const couchimport = require('couchimport');
+const dbname = 'sdcimages';
+
+const seed = async () => {
+	const db = nano.db.use(dbname);
+	try {
+		const info = await db.info();
+		console.log('DB Exists!', info);
+		await nano.db.destroy(dbname);
+	} catch (e) {
+		console.log('DB does not exist!');
+	}
+	await nano.db.create(dbname);
```

Comment:
does this block of code check if a db exists. and if does it destroys it and then in both cases it just creates an new db. Is my understanding correct?

Reply:
Yes that is correct.